### ISSUE TITLE
Seed nodes need to be hardcoded

### ISSUE THREAD

**Issue by jwill702** [ronpaulcoin/ronpaulcoin#1]
Seed nodes need to be hard coded so that users of the wallet do not need to install a conf file.

**Comment by ronpaulcoin** [comment]
I agree completely. That is one of the top requests. Next is the litecoin
tray icon :p

**Comment by jwill702** [comment]
For this to work properly we need a dns seed server.  This can be installed on any *nix cloud server.  If someone can install the server, I can install the code.

The other option is by changing the array on line
1252 in net.cpp

We would want a decent amount of nodes first though, which can be dumped from a seed server.

**Comment by ronpaulcoin** [comment]
Could we get one of the many mining pools to host this code that is needed
for the dns seed server? Most of them must be using some form of *nix
servers? And this would save us having to pay a monthly fee.

Colin
